docs/proposals/service-binding-specification/implementing-specification.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Implementing Service Binding Specification for Kubernetes
```

Comment:
Given that this doesn't describe implementation, rather describes how to deliver two sets of binding controller/CRDs, could you update the PR accordingly ?

( I've updated the title ..)

Reply:
I think the old title correctly identified what we're trying to do:  implement the spec.  The two sets of controllers / CRDs is a temporary state until current SBO users can be migrated to the spec implementation.

Reply:
The spec is not final. Parts of the spec (RC1) was being implemented when RC2 happened from June which is extremely distinct from RC1. 

If implementing the spec is the goal, I would say, let's wait for next year when it's stable and we've got enough feedback.

Reply:
Retitled to, "Proposal to add an alternate service binding implementation" because the content does say that the proposed implementation stays along with the existing one. 

Does this reflect the intent better? Also, I could also revert it to the original.

Reply:
IMHO, a better title could be "Support Service Binding Specification for Kubernetes"

Reply:
The intention of this proposal is really to suggest a way to support the spec. I think "Support Service Binding Specification for Kubernetes" summarizes the intention nicely.

Reply:
Sure sure!